### ISSUE TITLE
Sidebar quick links is missing for the aria-description page

### ISSUE THREAD

**Issue by s3812497** [mdn/content#23582]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description

### What specific section or headline is this issue about?

sidebar quick links

### What information was incorrect, unhelpful, or incomplete?

The sidebar that is usually located on the right-side of the screen is missing on this page.

### What did you expect to see?

The sidebar located on the right-side of the screen with quick links to the other accessibility HTML attributes.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-description`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-description/index.md
* Last commit: https://github.com/mdn/content/commit/f68449a8d7d8c04b1cdc90b22e780b202c4a1ff4
* Document last modified: 2022-09-24T08:39:35.000Z

</details>
